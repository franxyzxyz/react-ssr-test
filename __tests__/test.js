import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../src/components/index.js';


describe('Index (Snapshot)', () => {
  it('Index renders Index is me', () => {
    const component = renderer.create(<Index />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  })
})
