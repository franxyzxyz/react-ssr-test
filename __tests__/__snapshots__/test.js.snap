exports[`Index (Snapshot) Index renders Index is me 1`] = `
<html>
  <head>
    <meta
      charSet="utf-8" />
    <title>
      Test
    </title>
  </head>
  <body>
    <div
      id="content">
      <h1>
        INDEX is Me
      </h1>
      <a
        onClick={[Function]}
        style={Object {}}>
        PROFILE
      </a>
      <a
        onClick={[Function]}
        style={Object {}}>
        RANDOM
      </a>
    </div>
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      } />
    <script
      src="build/bundle.js" />
  </body>
</html>
`;
